roles.md

```diff
@@ -2,6 +2,8 @@
 
 **Table of Contents**
 - [Cultivator](#cultivator-aka-meeting-chair)
+- [Finance (External)](#finance-external)
+- [Finance (Internal)](#finance-internal)
```

Comment:
@hyphacoop/member-owners anyone has better names for these?

Reply:
Accounts Payable/Receivable (Bookkeeper? Analyst? Auditor?)
Payroll Coordinator

Reply:
Permaculture themed ones? 😄

Reply:
Finance (External) -> Financial Resource Coordinator
Finance (Internal) -> Worker Caretaker

Reply:
These feel pretty confusing/not informative of what they do (nor really capturing the permaculture/mycology spirit?) 

Can we bring a hsortlist fo the all hands and aim to get feedback there?

Reply:
Finance set a deadline to sort these names by weekend, so I am going with `Finance Steward` and `Payroll Coordinator`. We can make future revisions as needed. Don't think it's worthwhile to bring the naming into all hands.